client/src/components/changePasswordForm.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import toastr from 'toastr';
+import NavigationBar from './NavigationBar';
+import changePassword from '../actions/changePassword';
+import '../styles/passwordchange.scss';
+/**
+ *
+ *
+ * @class ChangePasswordForm
+ *
+ * @extends {Component}
+ */
+
+class ChangePasswordForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      oldPassword: '',
+      password: ''
+    }
+  }
+
+  onNameChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  onSubmit = (event) => {
+    event.preventDefault();
+    this.props.changePassword(this.state).then((res) => {
+      toastr.success('Password successfully changed')
+      this.setState({
+        oldPassword: '',
+        password: ''
+      })
+    })
+    .catch((error) => {
```

Comment:
### Nitpick

Since this is a single line statement, you might want to return #L39 to let the block know it has completed. 
It can also be refactored to look like this if you don't want to use the `return` keyword:
`.catch(error => toastr.error(error.response.data.error));`